.github/workflows/build.yml

```diff
@@ -19,11 +19,18 @@ jobs:
         with:
           persist-credentials: false
 
-      - name: Setup Platform 🛠️
+      - name: Setup Bun 🛠️
         uses: oven-sh/setup-bun@4bc047ad259df6fc24a6c9b0f9a0cb08cf17fbe5 # v2
         with:
           bun-version: 1.1.18
 
+      - name: Cache Bun dependencies 📦
```

Comment:
Is there a performance improvement to cache the dependencies? I'm asking that because initially I thought it would be the case, but then @lishaduck informed me, that the pipeline is even faster without the caching step: https://github.com/material-extensions/vscode-material-icon-theme/pull/2359#discussion_r1638415683

Would you mind showing some information that the caching brings some value here?

Reply:
> Is there a performance improvement to cache the dependencies? I'm asking that because initially I thought it would be the case, but then @lishaduck informed me, that the pipeline is even faster without the caching step: [#2359 (comment)](https://github.com/material-extensions/vscode-material-icon-theme/pull/2359#discussion_r1638415683)
> 
> Would you mind showing some information that the caching brings some value here?

Really, there is practically no difference 😐